lightwood/analysis/nc/wrappers.py

```diff
@@ -19,25 +18,6 @@ def clear_icp_state(icp):
         icp.normalizer.model = None
 
 
-def restore_icp_state(col, hmd, session):
```

Comment:
why do we remove this?

Reply:
Because it was a dead function, never used.